### DISCUSSION TITLE
导出png图片，如果使用了react节点，节点中还使用了img，这个img的内容会显示不出来

### DISCUSSION THREAD

**Discussion by luoxy1001** [antvis/X6#2224] [Q&A]
```
graph.toPNG(
      (dataUri: string) =>
        DataUri.downloadDataUri(dataUri, 'chart.png'),
      { serializeImages: true },
    );
```
已解决，先把图片转成base64

**Comment by czp709** [comment]
> ```
> graph.toPNG(
>       (dataUri: string) =>
>         DataUri.downloadDataUri(dataUri, 'chart.png'),
>       { serializeImages: true },
>     );
> ```
> 
> 已解决，先把图片转成base64

可以看一下你代码是怎样解决的吗?
